command/ReplCommand.java

```diff
@@ -294,6 +294,177 @@ public boolean isTransaction() {
         public Transaction asTransaction() {
             return this;
         }
+
+    }
+
+    class Options {
+
+        public static String token = "transaction-options";
+
+        static GraknOptions from(String[] optionTokens, boolean isCluster) {
+            if (isCluster) return parseClusterOptions(optionTokens, GraknOptions.cluster());
+            else return parseCoreOptions(optionTokens, GraknOptions.core());
+        }
+
+        private static GraknOptions.Cluster parseClusterOptions(String[] optionTokens, GraknOptions.Cluster options) {
+            for (int i = 0; i < optionTokens.length; i += 2) {
+                String token = optionTokens[i];
+                String arg = optionTokens[i + 1];
+                assert token.charAt(0) == '-' && token.charAt(1) == '-';
+                Option<GraknOptions.Cluster> option = Options.Cluster.clusterOption(token.substring(2));
+                try {
+                    options = option.build(options, arg);
+                } catch (IllegalArgumentException e) {
+                    throw new GraknConsoleException(e);
+                }
+            }
+            return options;
+        }
+
+        private static GraknOptions parseCoreOptions(String[] optionTokens, GraknOptions options) {
+            for (int i = 0; i < optionTokens.length; i += 2) {
+                String token = optionTokens[i];
+                String arg = optionTokens[i + 1];
+                assert token.charAt(0) == '-' && token.charAt(1) == '-';
+                Option<GraknOptions> option = Options.Core.coreOption(token.substring(2));
+                try {
+                    options = option.build(options, arg);
+                } catch (IllegalArgumentException e) {
+                    throw new GraknConsoleException(e);
+                }
+            }
+            return options;
+        }
+
+        static class Core {
+
+            static List<Option.Core> options = list(
+                    Option.core("infer", Option.Arg.BOOLEAN, "Enable or disable inference", (opt, arg) -> opt.infer((Boolean) arg)),
+                    Option.core("traceInference", Option.Arg.BOOLEAN, "Enable or disable inference tracing", (opt, arg) -> opt.traceInference((Boolean) arg)),
```

Comment:
I think it makes more sense to use kebab case: `trace-inference`, `batch-size`, and so on, right? in console, the UX is primarily kebab case: CLI options and grakn schema are in kebab case

Reply:
fine fineee

Reply:
done

Reply:
pizza